spyder/plugins/editor/widgets/editor.py

```diff
@@ -622,7 +622,17 @@ def show_in_external_file_explorer(self, fnames=None):
         """Show file in external file explorer"""
         if fnames is None:
             fnames = self.get_current_filename()
-        show_in_external_file_explorer(fnames)
+        try:
+            show_in_external_file_explorer(fnames)
+        except FileNotFoundError as error:
+            file = str(error).split("'")[1]
+            if "xdg-open" in file:
+                msg_title = _("Warning")
+                msg = _("Spyder can't show this file in the external file "
+                        "explorer because the <tt>xdg-utills</tt> package is "
+                        "not only available in your system.")

```

Comment:
```suggestion
                        "not available in your system.")
```

Sorry, i missed removing that word in my previous suggestion.

Reply:
I'd say "on your system" instead of "in your system" and it looks like there is one letter l too much in "xdg-utils"

Reply:
Thanks for the suggestions @jitseniesen!

Reply:
Thanks @jitseniesen